cypress/e2e/pages/testsForAllPages.js

```diff
@@ -28,6 +32,23 @@ export const testsThatAlwaysRunForAllPages = ({
       }
     });
 
+    it('should have href values correctly formed', () => {
+      const allowedUrls = ['programmes/p0703hz7', 'send/u94753086'];
```

Comment:
Can we annotate these URLs to explain where they come from?
```suggestion
      const allowedUrls = ['programmes/p0703hz7', // present on https://www.bbc.com/persian/topics/cw9qgeqd1zqt & redirects to https://www.bbc.com/persian/podcasts/p0703hz7
      'send/u94753086' // insert comment here
      ];
```